cmd/zoekt-webserver/main.go

```diff
@@ -459,13 +459,28 @@ Possible remediations:
 	}
 }
 
+func diskUsage(path string) (*disk.UsageStat, error) {
+	duPath := path
+	if runtime.GOOS == "windows" {
+		duPath = filepath.VolumeName(duPath)
+	}
+	usage, err := disk.Usage(duPath)
+	if err != nil {
+		return nil, fmt.Errorf("diskUsage: %w", err)
+	}
+	return usage, err
+}
+
 func mustRegisterDiskMonitor(path string) {
 	prometheus.MustRegister(prometheus.NewGaugeFunc(prometheus.GaugeOpts{
 		Name:        "src_disk_space_available_bytes",
 		Help:        "Amount of free space disk space.",
 		ConstLabels: prometheus.Labels{"path": path},
 	}, func() float64 {
-		usage, _ := disk.Usage(path)
+		// I know there is no error handling here, and I don't like it
```

Comment:
Tiny comment: these comments are unrelated to the change and not super useful (and also use "I" in an inaccurate way :))